### PR TITLE
Bump version of language-css

### PR DESCRIPTION
**Pull request by cssinate**
### Identify the Bug

https://github.com/atom/atom/issues/21954

### Description of the Change

Bumping version of language-css from 0.44.4 to 0.44.5

### Verification Process

All tests passed in language-css

### Release Notes

N/A

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4736,7 +4736,7 @@
       "integrity": "sha512-37RDzLiqSHJjopFg3T/Sqntx793OMQ4o7tkf3IuKe68Mshz3mwavJhIaIX/mca+0u/aWquySJFzlZ4D8EdtRSA=="
     },
     "language-css": {
-      "version": "https://www.atom.io/api/packages/language-css/versions/0.44.4/tarball",
+      "version": "https://www.atom.io/api/packages/language-css/versions/0.44.5/tarball",
       "integrity": "sha512-BBlnxUx9SCv+mIYOBWUApvyB/rSOewy2m8g6f3Vk2LfLf0+t8+Hcay5hWqXqpyKS4BUa15E3TVhl3PHEpqwZ1w==",
```
Comment (DeeDeeG) [inline package-lock.json:4740]: With the version bump for this package, I'd expect the SHA512sum in the "integrity" field to be updated as well.

You can run `npm install --package-lock-only` in the root of the atom repo to update the SHA512sum/integrity field.

**Comment by DeeDeeG** [comment]
I have a commit (https://github.com/DeeDeeG/atom/commit/99afd50d2dae61805142ab94ba75c0ec8b56e556) that fixes the `package-lock.json`. But there is now [a failing test](https://dev.azure.com/DeeDeeG/b/_build/results?buildId=1099&view=logs&j=c5d9a21a-e43e-5eaa-4a5a-044feacfb7bd&t=1bdeab21-3bba-5fef-2971-1d9f5fd6fff7&l=144), so there is some work left beyond that.

@cssinate I can make my own PR here at the main Atom repo to continue working on this, or I can open a PR at your fork to add just my commit to this PR's branch. I'll lean toward making my own PR here if no response.

**Comment by cssinate** [comment]
Either is fine with me. Whichever your preference is.

**Comment by DeeDeeG** [comment]
I opened https://github.com/atom/atom/pull/21963. You can subscribe on the sidebar of that issue if you want notifications of any updates with that. (Or you can deliberately _not do that_ to keep your inbox clean!!)

I do hope the failing test can be figured out and this can land land to fix the code highlighting. Thanks for getting the ball rolling!

**Comment by sadick254** [comment]
I'm going to close this since there is a similar PR to bump `language-css` package. As @DeeDeeG suggested you can subscribe to changes on  #21963.

**Pull request closed without merging**